packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -97,6 +97,9 @@ export interface InteriorSequencePlace {
 
 /**
  * Defines a side relative to a character in a sequence.
+ * If it is toward the other interval endpoint, it is exclusive.
+ * If it is away from the other interval endpoint, it is inclusive.
+ * E.g. start + Side.After is exclusive of the character.
```

Comment:
Maybe rather than toward and away, we could say nearer and farther? And I might move this documentation to `SequencePlace`, since we do already use `Side` in spatial queries like `OverlappingIntervalsIndex`. I like the reframing of exclusivity to be relative to the other endpoint, but I haven't come up with a way to word it so that it makes sense with other uses of `SequencePlace` or `Side`.

Reply:
What are the other uses? I don't see any, but maybe those are planned?

Reply:
https://github.com/microsoft/FluidFramework/blob/61cf470556a22faa60a3556c7c67631c5bee9bfe/packages/dds/sequence/src/intervalIndex/overlappingIntervalsIndex.ts#L51-L56

For example this method takes into account `Side` when doing spatial queries of overlapping endpoints. I see that `findOverlappingIntervals` doesn't do the same -- I'll take a look at that, I forget why that's the case.

Reply:
I made the changes you suggested.

I changed the wording to be more general than intervals to say if it's the endpoint of a range.